app/timetables/admin.py

```diff
@@ -97,7 +97,7 @@ class EventAdmin(TimeStampAdmin):
 class VendorAdmin(DefaultAdmin):
     """Admin customisation for Vendor model."""
 
-    fields = ('name', 'slug', 'info', 'start_date', 'end_date')
+    fields = ('name', 'slug', 'info')
```

Comment:
@andela-kanyanwu: No response or approval yet! Please, round up the review as soon as you have time. Thanks